dists/bookworm/main/binary-amd64/Packages

```diff
@@ -1,3 +1,19 @@
+Package: zen
+Version: 5.0.2
+Architecture: amd64
+Maintainer: Zen Blockchain Foundation <info@horizen.io>
```

Comment:
Cronic are we planning to change ZBF to Horizen Foundation at some point or not really?

Reply:
It would have to be changed on zen repo. I'll create a ticket.